tokio/src/runtime/driver.rs

```diff
@@ -80,6 +80,18 @@ impl Handle {
 
         self.io.unpark();
     }
+
+    cfg_time! {
+        /// Returns a reference to the time driver handle.
+        ///
+        /// Panics if no time driver is present
+        #[track_caller]
```

Comment:
Does this `#[track_caller]` actually reach all the way to the user anywhere? Can you update the panic location tests if necessary?

Reply:
Same as [here](https://github.com/tokio-rs/tokio/blob/master/tokio/tests/time_panic.rs#L73).